### ISSUE TITLE
wrong RDNN for dbus service

### ISSUE THREAD

**Issue by decathorpe** [elementary/contractor#30]
The dbus service file is still named `org.elementary.contractor.service`, everything is being moved to `io.elementary.*` namespace.

**Comment by drowsymike** [comment]
Hello! I'd like to work on this. Is it still relevant? I'm planning to rename the DBus service to `io.elementary.contractor` (in the file `data/org.elementary.contractor.service.in`) and update the references in other files in the `src/` directory. Should I do it for the whole repository in one PR? And one more question: should I rename name of the `org.elementary.contractor.service.in` file?

**Comment by ryonakano** [comment]
> Is it still relevant?
> Should I do it for the whole repository in one PR?
> And one more question: should I rename name of the `org.elementary.contractor.service.in` file?

Yes for 3 times, see #31. But we also need to update occurrences in other repositories: https://github.com/search?q=org%3Aelementary+org.elementary.Contractor&type=code

This would be a breaking change so it would be a good time to get this in OS 9.

**Comment by drowsymike** [comment]
I've opened a PR to rename the RDNN to io.elementary.contractor. I've verified that it built correctly and the service starts up (I built and executed the app in elementary OS 8). Looking forward to your review!

**Comment by ryonakano** [comment]
@leftboardcuttenstring Thanks for your PRs! In case you're missing, we also need to work in Calendar and Photos:

https://github.com/elementary/calendar/blob/44d2c0eb37da965fb2d5e55f2867cdcc238fb225/io.elementary.calendar.yml#L19

https://github.com/elementary/photos/blob/c85407b018d1aafd5bfa5acfda5634aeaa62d79d/io.elementary.photos.json#L16
